napari/_app_model/constants/_commands.py

```diff
@@ -75,7 +75,6 @@ class CommandId(str, Enum):
     LAYER_PROJECT_MEAN = 'napari:layer:project_mean'
     LAYER_PROJECT_MEDIAN = 'napari:layer:project_median'
 
-    @property
```

Comment:
This change has been made to make `title()` a method, consistent with `string` which is this classes supertype.

Since this is in an underscored file I assume it's private API and fine to change without deprecation?

Reply:
This is a little confusing to me as I feel `title()` on `str` does something different (returns a title-cased version of the string) from what is intended here (returning the "title" of the command). The name collision is unfortunate though.

Is it necessary for this class to inherit from `str`? Maybe it could use [StringEnum](https://github.com/napari/napari/blob/9bea15e7b89156654ae02c3dc3144d6fc244aab3/napari/utils/misc.py#L314) instead?

Reply:
How about renaming the property to `.command_title` here?

Reply:
I'm pretty sure there already is a discussion on this topic somewhere as I already encountered this issue.

I believe the answer was that we can't change as it's public API and might be used in npe2/napari hub ?

If change there is, i believe we'll need proper deprecation, i.e. add a `.command_title`, and emit a deprecation warning for some period of time.

Reply:
I'm not at my computer, but if I recall correctly this was strictly internal. Just a convenience.  So I would change the name if I were yoh

Reply:
Almost nothing about naparis usage of app model should be public api, unless that has changed since I worked on it

Reply:
Thanks ! My memory failed me then :-) 

+1 to rename then.

Reply:
👍 I changed the name, was easier to redo the commit so I force pushed away the original commit.